### PR TITLE
Added logging and zip'd all files in Azure, AWS, GCP Scripts

### PR DESCRIPTION
**Pull request by vpritamani-rubrik**

### CODE REVIEW

File: CLOUD/Get-AWSSizingInfo.ps1 [hunk 1]
```diff
@@ -893,4 +907,35 @@ Write-Host "Total storage capacity of all FSx volumes: $fsxTotalCapacityGiB GiB
...
+Compress-Archive -Path $outputFiles -DestinationPath $archiveFile
+
+# Remove the original files
+foreach ($file in $outputFiles) {
+    Remove-Item -Path $file -ErrorAction SilentlyContinue
+}
+
+Write-Host
+Write-Host
+Write-Host "Results have been compressed into $archiveFile and original files have been removed." -ForegroundColor Green
```
Comment (DamaniN) [inline CLOUD/Get-AWSSizingInfo.ps1:941]: @vpritamani-rubrik, please add an instructional line after this, telling the user to send the zip file to their Rubrik representative.

File: CLOUD/Get-AzureSizingInfo.ps1 [hunk 2]
```diff
@@ -1128,6 +1131,31 @@ Write-Host "Output files are:"
...
+Compress-Archive -Path $filePaths -DestinationPath $archiveFile
+
+# Remove the original files
+foreach ($file in $filePaths) {
+    Remove-Item -Path $file -ErrorAction SilentlyContinue
+}
+
+Write-Host
+Write-Host
+Write-Host "Results have been compressed into $archiveFile and original files have been removed." -ForegroundColor Green
```
Comment (DamaniN) [inline CLOUD/Get-AzureSizingInfo.ps1:1156]: @vpritamani-rubrik, please add an instructional line after this, telling the user to send the zip file to their Rubrik representative.

File: CLOUD/Get-GCPSizingInfo.ps1 [hunk 3]
```diff
@@ -163,3 +173,21 @@ Write-Host "Total capacity of all disks: $totalGiB GiB or $totalGB GB or $totalT
...
+Compress-Archive -Path $outputFiles -DestinationPath $archiveFile
+
+# Remove the original files
+foreach ($file in $outputFiles) {
+    Remove-Item -Path $file -ErrorAction SilentlyContinue
+}
+
+Write-Host
+Write-Host
+Write-Host "Results have been compressed into $archiveFile and original files have been removed." -ForegroundColor Green
```
Comment (DamaniN) [inline CLOUD/Get-GCPSizingInfo.ps1:193]: @vpritamani-rubrik, please add an instructional line after this, telling the user to send the zip file to their Rubrik representative.

**Review by DamaniN — CHANGES_REQUESTED** [review]
@vpritamani-rubrik, please add instructional lines, telling the user to send the zip file to their Rubrik representative.

**Comment by vpritamani-rubrik** [comment]
Added instructional lines to all files @DamaniN

**Review by DamaniN — CHANGES_REQUESTED** [review]
@vpritamani-rubrik while testing this I found that if I CTL-C the script that the transcript is still going. Everything else I do at the prompt is written to the output.log file. You'll need to find a way to capture break signals and exit the script gracefully by running Stop-Transcript first.

**Review by DamaniN — CHANGES_REQUESTED** [review]
@vpritamani-rubrik the Start-Transcript command doesn't appear to zero out the output.log file before writing to it. Instead, the file is appended to. You'll need to figure out if the output.log file exists already. If it does, you'll need to delete it.

**Comment by vpritamani-rubrik** [comment]
@DamaniN Only way I could find to handle break-signals and still Stopping Transcript is by putting the whole thing in a try-catch-finally, hence I implemented it like that. Also cleared existing log file if exists before writing to it as requested

**Review by DamaniN — COMMENTED** [review]
LGTM

**Review by DamaniN — APPROVED** [review]
LGTM

**Pull request merged**